### DISCUSSION TITLE
Breadcrumb structured data on pages with custom url?

### DISCUSSION THREAD

**Discussion by pensivedog** [gethyas/hyas#498] [Q&A]
My website has several content types whose content items are neatly arranged in directories like `content/content-type/leaf-bundle/`. No problem there. However, I have a bunch of pages that, for the sake of keeping my `content` directory organized, live in `content/pages/page-title/`. In the font matter of these pages, I set `url` manually so that the URLs don't include `/pages/`.

For all of these content items, Google Search Console gives the error `Either "name" or "item.name" should be specified (in "itemListElement")
Items with this issue are invalid`.

Any idea how I can fix this error for these pages?

**Comment by h-enk** [comment]
Confirmed — see also https://github.com/thuliteio/seo/issues/24

> Any idea how I can fix this error for these pages?

Not yet — I will be looking into this and get back to you
